src/Renci.SshNet/Sftp/SftpFileAttributes.cs

```diff
@@ -572,5 +612,37 @@ internal static SftpFileAttributes FromBytes(byte[] buffer)
                 return FromBytes(stream);
             }
         }
+
+        internal static DateTime ToLocalTime(DateTime value)
+        {
+            DateTime result;
+
+            if (value == DateTime.MinValue)
+            {
+                result = DateTime.SpecifyKind(DateTime.MinValue, DateTimeKind.Local);
+            }
+            else
+            {
+                result = value.ToLocalTime();
+            }
+
+            return result;
+        }
+
+        internal static DateTime ToUniversalTime(DateTime value)
```

Comment:
This method can be removed.

Reply:
I kept ToUniversalTime and made it private. It's still needed by the setters for the LastAccessTime and LastWriteTime properties. Since those are public properties in a public class, it's probable that some code somewhere calls them and passes in DateTime values with Kind == Local or Unspecified.